playbooks/roles/freeipa_server/tasks/install_freeipa_server.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: 'Get FreeIPA Shortname'
+  shell: hostname -s
+  register: freeipa_server_shortname
```

Comment:
Actually, I don't think I'm using this variable anywhere, so I'll just remove it.